lua/nvim-treesitter/configs.lua

```diff
@@ -25,9 +24,31 @@ local config = {
         scope_incremental="grc",
         node_decremental="grm"
       },
-      is_supported = function(lang)
-        return queries.get_query(lang, 'locals')
-      end
+      is_supported = queries.has_locals
+    },
+    refactor = {
+      highlight_definitions = {
+        enable = false,
+        disable = {},
+        is_supported = queries.has_locals
+      },
+      smart_rename = {
+        enable = false,
+        disable = {},
+        is_supported = queries.has_locals,
+        keymaps = {
+          smart_rename = "grr"
+        }
+      },
+      navigation = {
+        enable = false,
+        disable = {},
+        is_supported = queries.has_locals,
+        keymaps = {
+          goto_definition = "gnd",
+          list_definitions = "gnD"
```

Comment:
I'm not sure about these default bindings.  Open to any other suggestions.

Reply:
I like them. `grr` is super easy to remember :dog:. Whenever you don't like the name you go like `grrrrrr`